packages/react-experiments/src/components/SelectedItemsList/Items/TriggerOnContextMenu.tsx

```diff
@@ -8,7 +8,7 @@ export const TriggerOnContextMenu = <T extends any>(ItemComponent: React.Compone
     const { onTrigger } = props;
     // eslint-disable-next-line react-hooks/rules-of-hooks -- this is a component
     const trigger = React.useCallback(
-      e => {
+      (e: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
can we get some consistency here ?

any / vs explicit unused type 

<img width="1961" alt="image" src="https://github.com/user-attachments/assets/6ab8d9f2-9edc-4c33-a91d-3d237c2547f0" />